webextensions/api/browserAction.json

```diff
@@ -386,7 +386,11 @@
                 ]
               },
               "edge": {
-                "version_added": "14"
+                "version_added": "14",
+                "notes": [
+                  "This call is not persisted.",
+                  "The `imageData` paremeter is not accepted, `path` is requred"
```

Comment:
Let's clean up the formatting here a little. How about two notes, like this:

* `"The <code>imageData</code> parameter is not accepted."`
* `"The <code>path</code> parameter is required."`